rskj-core/src/main/java/org/ethereum/rpc/converters/CallArgumentsToByteArray.java

```diff
@@ -106,4 +106,17 @@ public byte[] gasLimitForGasEstimation(long gasCap) {
 
         return this.getGasLimit();
     }
+
+    public byte[] gasLimitForCall(long gasCap) {
+        byte[] gasLimitBytes = this.getGasLimit();
```

Comment:
```suggestion
        byte[] gasLimitBytes = this.getGasLimit();
        if (gasCap <= 0) {
            return gasLimitBytes;
        }
```
`gasCap` can be set to zero in a config

Reply:
Done